### DISCUSSION TITLE
Unable to Delete/Unlink Custom Domain from Vercel

### DISCUSSION THREAD

**Discussion by Phemel0** [vercel/community#3603] [Help] [answered]
### Summary

Hello, I deleted my Domain from a deployed project on Vercel and removed it from the custom domain settings, but I still receive the "404 not found" message when I access the domain. I have deleted the domain from both Vercel and my Web Server Records. How can I completely remove the domain from Vercel altogether?

### Steps to Reproduce

![vercelOne](https://github.com/vercel/community/assets/34798947/49a879e6-2d58-4413-a372-f8e79d3990a1)

**Comment by amyegan** [comment, marked as answer]
I suspect the domain has been removed from your Vercel project, but the A record or nameservers do not yet point at a new site. If you point the domain at a different site, then you should no longer see this error.

**Reply by Phemel0** [reply]
Hi @amyegan ,  

I 100% agree with you, even with the declared TTL some nameservers could have probably cached the DNS Records that point to Vercel for a slightly longer period (I'm assuming that is what happened). I did try out your suggestion as a final measure, pointed the domain elsewhere and waited for the new DNS records to propagate. Everything works as intended. 

Thanks for the suggestion and for reaching out.
